lib/tasks/lumen.rake

```diff
@@ -781,3 +738,90 @@ where works.id in (
     puts "#{date_time_task.call} Finishing the task"
```

Comment:
Is this for debugging purposes? Or we need it in the logs?

Reply:
I'm not sure of the original intent of this code. I note that there are a lot of puts statements in the rake task file; the should be converted to logging statements or removed. I will make a ticket for that (but please consider it outside the scope of this ticket).